mk/glib-tap.mk

```diff
@@ -35,6 +30,13 @@ check_PROGRAMS =
 check_SCRIPTS =
 check_DATA =
 
+### NOTE:
+### Pacemaker does not support installed tests, at least not of the unit test
+### variety.  References to the many installed_* variables have been removed
+### from this file but not from the next big comment.  Anywhere they are mentioned,
+### just remember we don't support that.  Unit tests are only run from the
+### source tree.
+
```

Comment:
Is the plan to update the big comment later? Should we rename the file?

Reply:
I didn't really have a bigger plan, but yeah I could do whatever.  I could continue stripping it down.  I don't think any of our unit tests use dist_test_data or test_data right now so that could all go away.  Renaming it would help reinforce that you shouldn't just grab an updated copy and dump it into the source tree.

Reply:
That sounds good

Reply:
Done.  Turns out we don't really need much of anything in that file in the first place.